### PR TITLE
feat: enable Urdu news - Ghost

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
Looks about what I would expect :thumbs_up:

How/when do you decide to remove the redirect to `/news` (english?

**Comment by raisedadead** [comment]
> How/when do you decide to remove the redirect to /news (english?

Which line are you talking about?

File: sites-enabled/10-www.freecodecamp.org.conf [hunk 1]
```diff
@@ -497,6 +502,55 @@ server {
     # include snippets/common/proxy-params.azure.conf;
   }
 
+  # reverse proxy client (urdu)
+  # TEMPORARY redirect to English learn
+  location /urdu {
+    return 302 https://www.freecodecamp.org/;
+  }
```
[the remark this answers is not in the document]
Comment (raisedadead) [reply sites-enabled/10-www.freecodecamp.org.conf:509]: Ah - these are currently there to capture and navigate users trying to get to localized learn (`/urdu` in this case should have a localized learn in Urdu) at some point. Until then we send them to English.

[the remark this answers is not in the document]
Comment (ShaunSHamilton) [reply sites-enabled/10-www.freecodecamp.org.conf:509]: Ah. I completely missed the fact that this is unrelated to /news 🤦‍♂️

**Pull request merged**